pbiviz.json

```diff
@@ -9,7 +9,7 @@
     "supportUrl": "http://community.powerbi.com",
     "gitHubUrl": "https://github.com/Microsoft/powerbi-visuals-mekkochart"
   },
-  "apiVersion": "1.13.0",
+  "apiVersion": "2.1.0",
```

Comment:
I do not think we need it. As far as I know, API version will be taken from `powerbi-visuals-api`

Reply:
it doesn't work in this way, tools still use pbiviz.json file.